src/test/resources/database/books/add-two-books-to-books-table.sql

```diff
@@ -0,0 +1,4 @@
+insert into books (id, title, author, isbn, price)
+values (2, 'Book 1unique', 'Author 1unique', 'ISBN 1unique', 200);
+insert into books (id, title, author, isbn, price)
+values (3, 'uniqueBook 1', 'uniqueAuthor 1', 'uniqueISBN 1', 200);
```

Comment:
You don't need to write an "insert" each time
```suggestion
insert into books (id, title, author, isbn, price)
values 
(2, 'Book 1unique', 'Author 1unique', 'ISBN 1unique', 200),
(3, 'uniqueBook 1', 'uniqueAuthor 1', 'uniqueISBN 1', 200);
```